airflow/providers/salesforce/hooks/salesforce.py

```diff
@@ -336,3 +348,44 @@ def object_to_df(
             df["time_fetched_from_salesforce"] = fetched_time
 
         return df
+
+    def write_schema_to_file(self, schema_file_name: str, obj: str, record_time_added: bool):
+        """
+        Export BigQuery schema to file
+
+        Write a JSON schema file with data types inferred from the Salesforce client.
+
+        :param schema_file_name: file name of the schema file (e.g schema.JSON)
```

Comment:
i'd suggest calling this parameter just filename or path

theres no other filenames it could be here and sometimes adding prefix in this scenario makes you second guess... is there something else i'm missing?